module_utils/ansible_nailgun_cement.py

```diff
@@ -308,6 +323,16 @@ def find_content_view_filter(module, name, content_view, failsafe=False):
     return handle_find_response(module, content_view_filter.search(), message="No content view filter found for %s" % name, failsafe=failsafe)
 
 
+def find_host(module, name, failsafe=False):
+    host = Host().search(set(), {'search': 'name="{}"'.format(name)})
+    return handle_find_response(module, host, message="No host found for %s" % name, failsafe=failsafe)
+
+
+def find_hostgroup(module, name, failsafe=False):
+    hostgroup = HostGroup(name=name).search(set(), {'search': 'name="{}"'.format(name)})
```

Comment:
Here's still the doubled `name` in search.

Reply:
fixed